Objects/dictobject.c

```diff
@@ -4157,10 +4534,20 @@ _PyDict_Contains_KnownHash(PyObject *op, PyObject *key, Py_hash_t hash)
     PyObject *value;
     Py_ssize_t ix;
 
+#ifdef Py_GIL_DISABLED
```

Comment:
This also looks like `contains_known_hash`

Reply:
Or does `contains_known_hash` look like this? 🤔